npm/@fastly/cli/package-helpers.js

```diff
@@ -0,0 +1,22 @@
+import { platform, arch } from "node:process";
+
+const knownPackages = {
+  "darwin arm64": "@fastly/cli-darwin-arm64",
+  "darwin x64": "@fastly/cli-darwin-x64",
+  "linux arm64": "@fastly/cli-linux-arm64",
+  "linux x64": "@fastly/cli-linux-x64",
+  "linux x64": "@fastly/cli-linux-386",
```

Comment:
```suggestion
  "linux x32": "@fastly/cli-linux-386",
```

Reply:
Fixed by removing this map and generating the package names dynamically.